packages/multichain-transactions-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/multichain-transactions-controller",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
Nit: do we need to release this controller at all ?

Reply:
Yes, it's required for the non-EVM/multichain initiative